packages/react-components/stories/circular-progress-bar.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { CircularProgressBar, CircularProgressBarProps } from '../lib/circular-progress-bar';
+import { Typography } from '@material-ui/core';
+
+export default {
+  title: 'Circular Progress Bar',
```

Comment:
Move this to a `Robots` section if `ProgressBar` and `CircularProgressBar` is moved to be next to `RobotInfo`.